.circleci/config.yml

```diff
@@ -89,7 +89,8 @@ commands:
     steps:
       - run:
           name: Install remote benchmark tool dependencies
-          command: TF_EXE_FILE_NAME=/workspace/terraform ./tests/benchmarks/remote/install_deps.sh
+          command: |
+            VERSION=0.14.8 ./deps/readies/bin/getterraform
```

Comment:
Having this in the CI configuration isn't ideal - it means you have to change CI to change the version. What if getterraform takes an optional option to a tfstate file? Then we could yank the version from it, and install that. Failing that, existing version in the file.

General rule: I want to get things out of circle configs, not into them - it ites us.